qutebrowser/qutebrowser.py

```diff
@@ -118,6 +118,9 @@ def get_argparser():
                        action='append')
     debug.add_argument('--qt-flag', help="Pass an argument to Qt as flag.",
                        nargs=1, action='append')
+    debug.add_argument('--debug-flag', type=debug_flag_error,
+                       help="Pass name of debugging feature to be turned on.",
+                       nargs=1, action='append')
```

Comment:
You'll also need to add a `dest='debug_flags'` here - otherwise you'd need to adjust the checks to do `... in self._args.debug_flag:` which reads a bit odd.